docs/adrs/jwt_token_caching.md

```diff
@@ -1,9 +1,9 @@
 <!-- SPDX-License-Identifier: MIT -->
 
 ---
-date: 2023-07-24
+date: 2023-12-04
```

Comment:
as discussed - remove this ADR record as well and bring it back once the PR with the Auth refactoring is ready to review